airflow/providers/databricks/hooks/databricks.py

```diff
@@ -408,3 +410,19 @@ def get_repo_by_path(self, path: str) -> Optional[str]:
                 raise e
 
         return None
+
+    def test_connection(self):
+        """Test the Databricks connectivity from UI"""
+        status, message = False, ''
+        hook = DatabricksHook(databricks_conn_id=self.databricks_conn_id)
+        try:
+            result = hook._do_api_call(endpoint_info=LIST_ZONES_ENDPOINT).get('zones', [])
+            if result:
+                self.log.info(result)
```

Comment:
Please correct me if I'm wrong.
This log won't be available in airflow UI or in the browser console. So since we are already raise exceptions in case of error which would be available in UI, this line does not add much value so either we should keep it as debug or remove it.

Reply:
Thats correct. It appears in the webserver container as shown below

`[2022-06-23 14:43:42,389] {databricks.py:421} INFO - ['us-east-2b', 'us-east-2c', 'us-east-2a'] [23/Jun/2022:14:43:42 +0000] "POST /api/v1/connections/test HTTP/1.1" 200 68 "http://localhost:8080/connection/edit/1" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36"`